text/intro.md

```diff
@@ -1,8 +1,6 @@
-#Introduction
+#Introduction to Text-Analysis
```

Comment:
Roger that… changed :-)
 <http://www.dato.com/>	 	Mark Enomoto
Designer 
mark@dato.com <mailto:mark@dato.com>	 	 <https://twitter.com/datoinc>	 <http://www.linkedin.com/company/graphlab>
 <https://www.facebook.com/datoinc>	 <http://www.youtube.com/channel/UC9T07WMdwB6IPf_dKXhVsCQ>



> On Aug 3, 2015, at 9:49 AM, Zach Nation <notifications@github.com> wrote:
> 
> In text/intro.md <https://github.com/dato-code/userguide/pull/20#discussion_r36123287>:
> 
> > @@ -1,8 +1,6 @@
> > -#Introduction
> > +#Introduction to Text-Analysis
> Why is this Introduction to Text-Analysis? The left nav says Text analysis (no dash, no "Introduction to", no capital letter on analysis). I think they should match.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/dato-code/userguide/pull/20/files#r36123287>.
>